Formula/spek.rb

```diff
@@ -0,0 +1,25 @@
+class Spek < Formula
+  desc "Acoustic spectrum analyser"
+  homepage "https://www.spek.cc"
+  url "https://github.com/alexkay/spek/releases/download/v0.8.5/spek-0.8.5.tar.xz"
+  sha256 "1bccf85a14a01af8f2f30476cbad004e8bf6031f500e562bbe5bbd1e5eb16c59"
+  license "GPL-3.0-only"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "ffmpeg"
+  depends_on "wxwidgets"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--libdir=#{lib}",
+                          "--disable-silent-rules"
```

Comment:
```suggestion
    system "./configure", *std_configure_args
```

Reply:
https://docs.brew.sh/Formula-Cookbook#install-the-formula
The Cookbook said "Some configure scripts do not recognise e.g. --disable-debug. If you see a warning about it, remove the option from the formula."

The configure script does not recognise --disable-debug. How can I remove this option?

Reply:
You could follow `apophenia` and use:
```
args = std_configure_args - ["--disable-debug"]
system "./configure", *args, "--disable-silent-rules"
```